PatternPal/PatternPal.Extension/Commands/SubscribeEvents.cs

```diff
@@ -119,6 +119,7 @@ public static async Task UnsubscribeEventHandlersAsync()
             _dteSolutionEvents.BeforeClosing -= OnSolutionClose;
             _dteDebugEvents.OnEnterBreakMode -= OnExceptionUnhandled;
             _dteDebugEvents.OnEnterDesignMode -= OnDebugProgram;
+            _dteDocumentEvents.DocumentSaved -= OnDocumentSaved;
```

Comment:
If I am looking at this correctly, you also forgot to subscribe/unsubscribe for `OnFileCreate` now, right?

Reply:
No, OnFileCreate is removed when the watcher is removed.